packages/core/admin/admin/src/components/AutoReloadOverlayBlocker/Overlay.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
+
+// TODO refactor with DS
+const Overlay = styled.div`
```

Comment:
Yes this component is used to prevent user interactions when the server is restarting:
- Used in the ctb
- Used in the Marketplace
- Used in the Plugins page

It's a very important one, similarly to the Overlay component that we use to prevent user navigation when performing requests.